pages/index.vue

```diff
@@ -106,6 +110,17 @@ export default {
       },
     };
   },
+  async mounted() {
+    if (this.$config.isFederationAPI) {
+      this.isFederationAPI = this.$config.isFederationAPI;
+
+      const response = await this.$axios.get(`${this.$config.apiQueryURL}nodes/`);
+      this.nodes = response.data.reduce((acc, node) => ({
```

Comment:
```suggestion
      this.nodes = response.data.reduce((temp_node_array, node) => ({
```
or something less cryptic